app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,31 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+  before_action :load_user, only: %i[create]
+  before_action :load_relationship, only: %i[destroy]
+
+  def create
+    current_user.follow(@user)
```

Comment:
note to reseacrh: so sánh scope và class method

Reply:
dạ a